recipes/imgui/all/conanfile.py

```diff
@@ -90,7 +110,8 @@ def package(self):
         cmake.install()
 
     def package_info(self):
-        self.conf_info.define("user.imgui:with_docking", bool(self._match_docking_branch()))
+        self.conf_info.define("user.imgui:with_docking", bool(self._is_docking_branch()))
+        self.conf_info.define("user.imgui:enable_test_engine", bool(self.options.get_safe("enable_test_engine")))
```

Comment:
This is an usual one (ignoring the docking one) - consumers can already check downstream with `self.dependencies['imgui'].options.enable_test_engine`
is there a scenario where one may want to build imgui with `enable_test_engine=True` but _also_ define `user.imgui:enable_test_engine` to False? (as that can be defined outside the recipe, independently)